bindings/rust/README.md

```diff
@@ -5,26 +5,25 @@ Rust bindings for the [Unicorn](http://www.unicorn-engine.org/) emulator with ut
 Checkout Unicorn2 source code at [dev branch](https://github.com/unicorn-engine/unicorn/tree/dev).
 
 ```rust
```

Comment:
Could you also add a README for `unicorn-engine-sys`? An example would be preferable, or simply a link to our github repo.

Reply:
Sure